app/assets/javascripts/application.js

```diff
@@ -11,18 +11,6 @@ String.prototype.capitalizeFirstLetter = function () {
     return this.charAt(0).toUpperCase() + this.slice(1);
 };
 
-// function used to remove fields in the datasets form
```

Comment:
This functionallity is no longer needed?

Reply:
It is no longer required indeed :) This has been replaced by functionality in the Vue components. I'll remove the comment